AzureCalling/Controllers/CallViewController.swift

```diff
@@ -253,19 +242,15 @@ class CallViewController: UIViewController, UICollectionViewDelegate, UICollecti
     }
 
     @IBAction func contentViewDidTapped(_ sender: UITapGestureRecognizer) {
-        if callingContext.callingInterfaceState == .connected {
-            infoHeaderView.toggleDisplay()
-        }
+        infoHeaderView.toggleDisplay()
     }
 
     private func onJoinCall() {
-        NotificationCenter.default.addObserver(self, selector: #selector(onRemoteParticipantsUpdated(_:)), name: .remoteParticipantsUpdated, object: nil)
-        NotificationCenter.default.addObserver(self, selector: #selector(recordingActiveChangeUpdated(_:)), name: .onRecordingActiveChangeUpdated, object: nil)
-        NotificationCenter.default.addObserver(self, selector: #selector(onCallStateUpdated(_:)), name: .onCallStateUpdated, object: nil)
-        onCallStateUpdated()
+        NotificationCenter.default.addObserver(self, selector: #selector(onRemoteParticipantsUpdated(_:)), name: NSNotification.Name(rawValue: "RemoteParticipantsUpdated"), object: nil)
+
+        infoHeaderView.toggleDisplay()
         meetingInfoViewUpdate()
         initParticipantViews()
-        activityIndicator.stopAnimating()
```

Comment:
I think we can still keep the indicator. There will be less conflicts when we merge later on.